improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -469,6 +508,113 @@ def test_calibrating_without_single_value_bins(self, warning_list=None):
         assert_allclose(result[1].data, expected_1)
         self.assertFalse(warning_list)
 
+    def test_calibrating_point_by_point(self, warning_list=None):
```

Comment:
I think the the `warning_list` argument and the assertion that the `warning_list` is False can be removed, as nowadays, following the introduction of the use of pytest, we're no longer testing for warnings. 

I did have a go at including the ManageWarnings decorator on the three added unit tests (test_calibrating_point_by_point, test_calibrating_spot and test_calibrating_spot_point_by_point) but although they all seem to generate warnings, only one of the tests (test_calibrating_point_by_point) actually seems to fail with a warning. I'm not sure why the behaviour is like this, but I think that the usage of the ManageWarnings decorator is really deprecated as we [disable warnings with pytest](https://github.com/metoppv/improver/blob/master/pytest.ini#L2). See https://github.com/metoppv/mo-blue-team/issues/437.